scripts/setup-ubuntu.sh

```diff
@@ -108,6 +108,11 @@ PACKAGES+=" composer"
 
 # Needed by package rust.
 PACKAGES+=" libssl-dev" # Needed to build Rust
+PACKAGES+=" llvm-12-dev"
+PACKAGES+=" llvm-12-tools"
+PACKAGES+=" clang-12"
+
+# Formerly needed by package rust. Not sure if needed by other packages.
 PACKAGES+=" clang-10"
```

Comment:
Yes, [Swift still needs this](https://github.com/termux/termux-packages/blob/master/packages/swift/build.sh#L107), I will try to remove it when I update to 5.5.3 this weekend.